aten/src/ATen/gen_backend_select_register.py

```diff
@@ -2,6 +2,7 @@
 # We process only those factory functions that have 'backend_select' flag in its native_functions.yaml definition.
 
 from code_template import CodeTemplate
+import tensor_options_utils as TOUtils
```

Comment:
I think it would be clearer if you just directly typed out `tensor_options_utils.blahblah`